docs/local_development.md

```diff
@@ -45,49 +43,140 @@ yarn install --force --frozen-lockfile
+
+# optionally run other triggers as needed, specify party name through variable
+$ make start-clearing-trigger party=Ccp
+$ make start-matching-engine party=Exchange
+$ make start-settlement-trigger party=Bank
+
+# optionally run a single auto approve trigger at a time
+$ make start-autoapprove party=AnyPartyName
 ```
 
-This will spin up the Daml Sandbox, run the automation bots, and run a yarn server for the frontend.
+Each trigger also writes to a log file within the `.dev` directory. If an error occurs during trigger
+execution, look there.
 
-The Daml Sandbox will automatically be bootstrapped with contracts from `daml/Setup.daml`.
+You may run multiple instances of the same trigger with different parties - each instance will get its own log.
 
-If you would like to change which Daml Script file is run when the Daml Sandbox starts, you can modify the `init-script` field in `daml.yaml`.
+### Stop the Triggers
 
-To access the frontend, point your browser to `http://localhost:3000`. To view the Daml ledger state in the Daml Navigator, point your browser to `http://localhost:7500`.
+You may explicitly stop any trigger by calling its symmetric `stop-*` make target on it. For example
+
+```sh
+$ make stop-clearing-trigger party=AnyPartyName
+$ make stop-matching-engine party=AnyPartyName
+$ make stop-settlement-trigger party=AnyPartyName
+$ make stop-autoapprove party=AnyPartyName
+```
 
-To stop running background processes:
+To stop all the running auto approve triggers in one go, run
 
+```sh
+$ make stop-autoapprove-all
 ```
-# stop the Daml server
-make stop_daml_server
 
-# stop all bots
-make stop_bots
+Be aware that stopping a trigger will delete the log file associated with it.
+
+Additionally, the trigger processes will die if the main sandbox process is killed,
+which may leave behind lingering files in `.dev`.
+
+### Start the UI
+
+The UI can be run locally through the webpack dev server. Simply
 
-# if you are using the included matching engine:
-make stop_matching_engine
+```sh
+$ cd ui
+$ yarn start
 ```
 
-Alternatively to running the Daml server in the background, `daml start` can be run in a separate terminal window.
+This should result in a UI on `localhost:3000`. Locally, files are watched for changes and the server
+should reload automatically.
+
+## Exberry Integration
+
+The Exberry integration can be started up locally. There are two components: the adapter, included
+in this repository, and the integration itself which is in its own repository.
+
+1. Clone [`daml-dit-integration-exberry`](https://github.com/digital-asset/daml-dit-integration-exberry) in a sibling directory next to `da-marketplace/`
+2. Open two terminals, one in the `da-marketplace/` directory, and one in the `daml-dit-integration-exberry/` directory.
+3. In `daml-dit-integration-exberry/`, create a file in the repo root named `int_args.yaml`. Fill it with the template below, replaced with your Exberry account details.
+
+```yaml
+"metadata":
```

Comment:
`clientId` and `tokenUrl` are no longer necessary

Reply:
thanks!